tests/apidoc_login_test.go

```diff
@@ -0,0 +1,41 @@
+package tests
+
+import (
+	"encoding/json"
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/match"
+	"github.com/matrix-org/complement/internal/must"
+	"github.com/tidwall/gjson"
+	"testing"
+)
+
+func TestLogin(t *testing.T) {
+	deployment := Deploy(t, "login", b.BlueprintAlice)
+	defer deployment.Destroy(t)
+	unauthedClient := deployment.Client(t, "hs1", "")
+	t.Run("parallel", func(t *testing.T) {
+		t.Run("POST /login can login as user", func(t *testing.T) {
```

Comment:
```suggestion
		// sytest: POST /login can log in as a user
		t.Run("POST /login can login as user", func(t *testing.T) {
```
As you are converting a sytest, please add the comment of the form `// sytest: the test name`. More information on this is [here](https://github.com/matrix-org/complement/blob/master/ONBOARDING.md#how-should-i-name-the-test-files--test-functions).